service/src/main/java/uk/gov/hmcts/reform/fpl/service/HearingBookingService.java

```diff
@@ -59,4 +59,8 @@ public HearingBooking getHearingBookingByUUID(List<Element<HearingBooking>> hear
             .findFirst()
             .orElse(null);
     }
+
+    public void filterPreviousHearings(List<Element<HearingBooking>> before, List<Element<HearingBooking>> after) {
```

Comment:
Change the variable names to `previous` and `all`, bit clearer than before and after

Reply:
personally i would change signature to List<Element<HearingBooking>> getChangedHearing(before, after){
return after.stream().filter(hearing -> !before.contains(hearing)).collect(toList());
}
I think it is a bit better as it does not  mutate original collection and name gives a bit more information what is happening

Reply:
happy to change to more verbose. As midevent it was not persisting changes to data across the lifetime of the event, but does make sense for readability

Reply:
@lewisbirks I think before and after make sense in terms of the method, previous and all, I think they're a bit amiguous. Any further ideas?